.github/workflows/release.yml

```diff
@@ -89,39 +91,22 @@ jobs:
           # `target`: Rust build target triple
           # `platform` and `arch`: Used in tarball names
           # `svm`: target platform to use for the Solc binary: https://github.com/roynalnaruto/svm-rs/blob/84cbe0ac705becabdc13168bae28a45ad2299749/svm-builds/build.rs#L4-L24
-          - runner: Linux-20.04
-            target: x86_64-unknown-linux-gnu
-            svm_target_platform: linux-amd64
-            platform: linux
-            arch: amd64
-          - runner: Linux-20.04
+          - runner: ubuntu-24.04
             target: x86_64-unknown-linux-musl
             svm_target_platform: linux-amd64
             platform: linux
             arch: amd64
-          - runner: Linux-20.04
-            target: aarch64-unknown-linux-gnu
-            svm_target_platform: linux-aarch64
-            platform: linux
-            arch: arm64
-          - runner: Linux-20.04
-            target: aarch64-unknown-linux-musl
-            svm_target_platform: linux-aarch64
-            platform: linux
-            arch: arm64
-          # This is pinned to `macos-13-large` to support old SDK versions.
-          # If the runner is deprecated it should be pinned to the oldest available version of the runner.
-          - runner: macos-13-large
+          - runner: macos-14
             target: x86_64-apple-darwin
             svm_target_platform: macosx-amd64
             platform: darwin
             arch: amd64
-          - runner: macos-latest-large
+          - runner: macos-14
             target: aarch64-apple-darwin
             svm_target_platform: macosx-aarch64
             platform: darwin
             arch: arm64
-          - runner: Windows
+          - runner: windows-2022
```

Comment:
is it needed to rename this?

Reply:
I do not see Windows in our GH hosted runners, are you sure that it will work?